assets/i18n/ar.json

```diff
@@ -566,7 +566,7 @@
 	"I18N_SIGNUP_ERROR_USERNAME_WITH_ADMIN": "أسماء المستخدمين التي بها 'admin' محجوزة.",
 	"I18N_SIGNUP_ERROR_USERNAME_WITH_SPACES": "يُرجَى التأكد من اسم المستخدم الخاص بك لا يحتوي على مسافات.",
 	"I18N_SIGNUP_FIELD_REQUIRED": "هذا الحقل مطلوب.",
-	"I18N_SIGNUP_LICENSE_NOTE": "تجدر الإشارة إلى أنه بالموافقة على شروط الاستخدام، فإنك توافق على أن أي محتوى أو تبرعات تقوم بها على موقعنا ستكون مرخصة بموجب المشاع الإبداعي النسبة-المشاركة بالمثل v.4.0، مع التنازل عن شرط الإسناد، يُرجَى مراجعة <a href=\"/terms\">شروط استخدام</a> موقعنا للحصول على مزيد من المعلومات حول منحة الترخيص، لمزيد من المعلومات حول المشاع الإبداعي النسبة-المشاركة بالمثل; انقر على <a href=\"#\" ng-click=\"$ctrl.showLicenseExplanationModal()\">هنا </a>.",
```

Comment:
By design angular doesnot allow dynamically interpolated HTML to call component's function.
So, I had to come up with a different solution to tackle this.